pkg/controller/pxc/users.go

```diff
@@ -233,6 +233,10 @@ func (r *ReconcilePerconaXtraDBCluster) handleRootUser(ctx context.Context, cr *
 	}
 	log.Info("Password updated", "user", user.Name)
 
+	if err := r.updateMySQLInitFile(ctx, cr, internalSecrets, user); err != nil {
```

Comment:
Should we do the same for MySQL 5.7 (`pgk/controller/pxc/users_witout_db.go`)?

Reply:
we do not have PITR for 5.7 at all

Reply:
Ok, got it.

Reply:
actually this is not about PITR anymore. it's for restoring a backup containing different passwords. I think @inelpandzic's comment is valid and I'll check